tests/python/pants_test/task/test_task.py

```diff
@@ -149,6 +150,10 @@ def execute(self):
     pass
 
 
+class TaskWithTargetFiltering(DummyTask):
+  options_scope = 'task-with-target-filtering'
+  target_filtering_enabled = True
+
```

Comment:
Two blank lines between top-level functions and classes - hoefully this fails lint in CI?

Reply:
[Yup](https://travis-ci.org/pantsbuild/pants/jobs/498374238), looks like I missed that check locally before pushing - my mistake, fix inbound